test/jdk/java/awt/im/4959409/bug4959409.java

```diff
@@ -77,19 +76,18 @@ public void windowOpened(WindowEvent e) {
             });
 
             jTextField.addKeyListener(new KeyAdapter() {
-
                 @Override
                 public void keyPressed(KeyEvent keyEvent) {
                     super.keyPressed(keyEvent);
                     int code = keyEvent.getKeyCode();
                     int mod = keyEvent.getModifiersEx();
                     if (code == '1' && mod == KeyEvent.SHIFT_DOWN_MASK) {
                         keyPressedEventLatch.countDown();
-                        jLabel.setText("keyPressed received for Shift+1");
-                        System.out.println("keyPressed received for Shift+1");
+                        jLabel.setText("keyPressed receive for Shift+1");
+                        System.out.println("keyPressed receive for Shift+1");
```

Comment:
Here, it was correct: “received”.

Reply:
Over curiously did search and replace . Corrected